Src/IronPython/Runtime/Binding/PythonProtocol.Operations.cs

```diff
@@ -1881,10 +1886,6 @@ public ItemBuilder(DynamicMetaObject/*!*/[]/*!*/ types, Callable/*!*/ callable)
             switch (op) {
                 case PythonOperationKind.Equal: name = nameof(PythonOps.CompareTypesEqual); break;
                 case PythonOperationKind.NotEqual: name = nameof(PythonOps.CompareTypesNotEqual); break;
-                case PythonOperationKind.GreaterThan: name = nameof(PythonOps.CompareTypesGreaterThan); break;
```

Comment:
Since you're playing with this code, consider making this function to a static local function in `MakeFallbackCompare`?

Reply:
It doesn't seem to show as updated, but this should now be a local method.